src/components/notifications/Whapi.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="mb-3">
+        <label for="whapi-api-url" class="form-label">{{ $t("API URL") }}</label>
+        <input id="whapi-api-url" v-model="$parent.notification.whapiApiUrl" type="text" class="form-control" required>
+    </div>
+
+    <div class="mb-3">
+        <label for="whapi-auth-token" class="form-label">{{ $t("Token") }}</label>
+        <input id="whapi-auth-token" v-model="$parent.notification.whapiAuthToken" type="text" class="form-control" required>
+    </div>
+
+    <div class="mb-3">
+        <label for="whapi-to-number" class="form-label">{{ $t("Phone number") }}</label>
+        <input id="whapi-to-number" v-model="$parent.notification.whapiToNumber" type="text" pattern="^[\d-]{10,31}$" class="form-control" required>
```

Comment:
idk if 10 is a small enough choice:

https://stackoverflow.com/questions/14894899/what-is-the-minimum-length-of-a-valid-international-phone-number

```suggestion
        <input id="whapi-to-number" v-model="$parent.notification.whapiToNumber" type="text" pattern="^[\d-]{8,31}$" class="form-control" required>
```

Reply:
The API docs say that the phone number should be between 10 and 31 digits long, I copied that

![immagine](https://github.com/louislam/uptime-kuma/assets/32812884/5b3417b9-f53f-4425-92b8-82175a004928)